exp-player/addon/components/exp-card-sort/component.js

```diff
@@ -168,6 +169,12 @@ export default ExpFrameBaseComponent.extend({
         }
         return responses;
     }).volatile(),
+
+    allowNext: Ember.computed('cards.[]', function() {
+        console.log(this.get('cards').length);
```

Comment:
Please remove `console.log` statements before committing

Also, I notice the indentation is different between these two lines?